db/seeds/code_lookups.rb

```diff
@@ -48,5 +48,6 @@
   { from: 'PP_SUNLIFEOC', to: 'Sunlife Financial of Canada' },
   { from: 'PP_ALLIEDDUN', to: 'Allied Dunbar' },
   { from: 'PP_ZURICH', to: 'Zurich' },
-  { from: 'PP_AXA', to: 'AXA' }
+  { from: 'PP_AXA', to: 'AXA' },
+  { from: 'PP_CANNLINC', to: 'Cannon Lincoln' }
```

Comment:
This doesn't seem related to the overall gist of this PR. Is that an oversight / accident?